Dockerfile

```diff
@@ -72,8 +75,22 @@ RUN touch src/canister_tests/src/lib.rs
 RUN touch src/metrics_encoder/src/lib.rs
 RUN npm ci
 
-RUN ./scripts/build
+RUN ./scripts/build --single-canister internet_identity
 RUN sha256sum /internet_identity.wasm
 
-FROM scratch AS scratch
-COPY --from=build /internet_identity.wasm /
+FROM deps as build_archive
```

Comment:
out of curiosity, do you know if the two builds run in parallel?

Reply:
No, because docker will build only one target at a time.